### ISSUE TITLE
Publish repository License

### ISSUE THREAD

**Issue by pbonnell** [benetech/ServiceNet#1767]
**User Story**\
As a social service organization developer I want to be able to implement a ServiceNet solution for a curently active resource repository.

**Description:**
The social service software space is crowded, competitive, and occasionally aggressive.
Is a given opportunity a clear path to sharing data in the open light of our utopian dream, or a veiled attempt to further encircle the commons in the clutches of private equity?
We see a beatiful garden, but fear it's unknown walls.
TO THE LICENSE, sisters - we will find our path cut clear.
TO THE LICENSE, brothers - our hopes fullfilled, our work laid bare.

**User Story**
(encompasses persona, purpose, value) - e.g. As a [persona] i want to be able to [purpose] so that I can [value]:

As a github user or member of the general public, I want to be able to know the licensing terms of the ServiceNet project.

**Description**

**Scope**
Published code / documentation.

**Acceptance criteria**
(includes edge cases)  - e.g. Given I am [user] I want to [action] so that [goal of task]:
*
Given I am a ServiceNet core developer I want to encourage other developers to evaluate and calloborate with me, my team, and Benetech.
 
**Dependencies**
(links to other related tasks):
*

- [Spring](https://github.com/spring-projects/spring-framework/blob/main/LICENSE.txt) (Apache 2)
- Java
- ??? to review

Any required **Analytics** for task:
*

Dev Signoffs
- [ ] Argue about definition of Open Source
- [ ] Ask [@greggish](https://github.com/openreferral/specification#license) his opnion
- [ ] Deeply meditate on professional and spiritual goals

QA Signoffs
- [ ] License is legible
- [ ] License is referenced in README
- [ ] Legal says "Do whatever, we don't care."

Product Signoff
- [ ] Approved for release

**Comment by greggish** [comment]
I agree, there should be a license! Would suggest Apache 2.0 although curious to hear if someone has other opinions.
